public_app/components/Details.js

```diff
@@ -0,0 +1,36 @@
+import React    from 'react';
+
+class Details extends React.Component {
```

Comment:
has only `render` function so can just be an arrow function that react also can render 
```
const Details = props => (
...
);
```

https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc